### PR TITLE
4/5 elijah

### PR DESCRIPTION
**Pull request by Eirizarr**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,7 +9,12 @@
 </head>
 <body>
   <header>
+<<<<<<< HEAD
+    <h1><a href="" rel="home"> Dominator Flight Booking</a></h1>
+      <!-- nav ?-->
+=======
```
Comment (abigailboyer) [inline index.html:15]: Remove your merge lines like <<<<<HEAD and ====== from all of your files that have them. index.html has them at 12, 15, 17, 84, 88 and 91

File: index.html [hunk 2]
```diff
@@ -76,7 +81,13 @@ <h2>Find a flight!</h2>
       </form>
     </section>
   </main>
+<<<<<<< HEAD
+  <footer>
+    <p> @Copyright Dominator Airlines!.......#DominateFlights.....</p>
+  </footer>
```
Comment (abigailboyer) [inline index.html:87]: stolley told us to stay away from adding a bunch of stuff all over the place, so we don't need a footer to declare some copyright that we don't have. so dont put footers anywhere and remove this one

File: screen.css [hunk 3]
```diff
@@ -31,6 +31,156 @@ body {
...
+
+input[type="submit"] {
+  color: #E4EEFB;
+  background-color: #5375A1;
+  cursor: pointer;
+  padding: 0.611em;
+  border-radius: 1em;
+  float: inherit;
+}
+=======
```
Comment (abigailboyer) [inline screen.css:150]: your html starts repeating here because you saved in the middle of a merge request or something like that. the top half should be deleted, the bottom half beneath the `======`  should get kept

File: screen.css [hunk 4]
```diff
@@ -31,6 +31,156 @@ body {
   line-height: 20px;
 }
 
+#Departflights ul {
+  display: inline-block;
+}
+
+#ReturnFlights ul{
+  display: inline-block;
+}
```
Comment (abigailboyer) [inline screen.css:40]: the order that css files ought to be in for the most readability is general, full body styles at the top, and then styles slowly increasing in specificity as you progress downwards through the file. so the first tag on the styles page should be `html {}` and then something like `returnflights ul {} ` would be nearer the bottom, in its own particular section. so i would suggest moving this. if you want me to show you where exactly, i can. 
_however_, this whole portion of your css file i'd actually recommend deleting because the bottom half is my code and the top seems like a straight duplicate. see my other comment below this one

File: searchresults/index.html [hunk 5]
```diff
@@ -4,17 +4,14 @@
   <title>Flight Booking: Search Results</title>
   <meta charset="utf-8" />
   <meta name="viewport" content="width=device-width,initial-scale=1.0,shrink-to-fit=no" />
+  <link rel="stylesheet" href="screen.css" />
```
Comment (abigailboyer) [inline searchresults/index.html:7]: remove this line, the css is already listed at line 9 and this file path is wrong

File: searchresults/index.html [hunk 6]
```diff
@@ -4,17 +4,14 @@
...
 </head>
 <body>
   <main>
     <h1>Search Results</h1>
-    <!-- todo: add form that replicates inputs from
-    the first form and allows editing -->
-
-
-  <!--  <form id="userform" action="#null" method="get">
+    <form id="Fligtupdate" action="#null" method="get">
```
Comment (abigailboyer) [inline searchresults/index.html:14]: this is unimportant but all of your ids should be lowercase so it'd be "flightupdate" not "Flightupdate"

File: searchresults/index.html [hunk 7]
```diff
@@ -44,40 +41,37 @@ <h1>Search Results</h1>
           <p id="confirmation"> ..................</p>
```
Comment (abigailboyer) [inline searchresults/index.html:41]: the form on this page is supposed to "replicate the form on the first page" like my comment says. it does not do that. this form is completely different. I had it commented out because it wasn't what it was supposed to be. The form on the first page is already done and ready all you have to do is copy it basically. I made the form and it has all the correct things we need on it, and all you gotta do is just copy and paste. the form that you have has things like cities to select from, but we're not doing that yet, right now we're just going to have them write in their own city because we're trying to keep everything as basic as possible until the actual forms work. 
so, lines 14-45 need to be deleted and replaced with the form i made on page 1.

File: searchresults/index.html [hunk 8]
```diff
@@ -44,40 +41,37 @@ <h1>Search Results</h1>
...
         </ul>
-      </li>
 
-      <li>
-        <label for="flighttwo">Flight Two</label>
-        <input type="radio" id="flighttwo" name="departflight" />
+  <!-- put the p inside of the value attribute instead??? -->
         <ul>
+          <li>
+          <label>Flight Two:<input type="radio" name="departflight" id="flighttwo"/></label>
```
Comment (abigailboyer) [inline searchresults/index.html:73]: i'm not sure why exactly you changed this, but the way i had it before was correct. you put the input button inside of its own label and i think you did it because you're trying to get it all on the same line but i'm pretty sure that's not semantically correct so please put it back the way i had it before because there are other ways to make the things go on the same line without using potentially-wrong html

File: searchresults/index.html [hunk 9]
```diff
@@ -94,12 +88,10 @@ <h2>Departing Flight</h2>
...
           </li>
         </ul>
-      </li>
-
-      <li>
-        <label for="flightthree">Flight Three</label>
-        <input type="radio" id="flightthree" name="departflight" />
         <ul>
+          <li>
+            <label>Flight Three:<input type="radio" name="departflight" id="flightthree"/></label>
```
Comment (abigailboyer) [inline searchresults/index.html:93]: same thing as above

File: seats/index.html [hunk 10]
```diff
@@ -4,8 +4,11 @@
   <title>Flight Booking: Seat Selection</title>
   <meta charset="utf-8" />
   <meta name="viewport" content="width=device-width,initial-scale=1.0,shrink-to-fit=no" />
+<<<<<<< HEAD
+=======
```
Comment (abigailboyer) [inline seats/index.html:8]: gotta get rid of all of these merge lines

File: seats/index.html [hunk 11]
```diff
@@ -4,8 +4,11 @@
...
   <link rel="stylesheet" href="http://basehold.it/20" />
 <<<<<<< HEAD
+>>>>>>> 9ae1c975c6216a7c4fe431c7f3d4f503333733ce
   <link rel="stylesheet" href="screen.css" />
```
Comment (abigailboyer) [inline seats/index.html:12]: the correct css filepath is the "../screen.css" so delete the one on line 12

File: seats/index.html [hunk 12]
```diff
@@ -37,8 +40,12 @@ <h2>Seat Selection</h2>
         </li>
       </ol>
   </selection>
+<<<<<<< HEAD
+
+=======
   <a href="../traveler/index.html"><button>Next</button></a>
   <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
   <script src="../site.js"></script>
+>>>>>>> bdc99a185ba6dd7ce41c81e3da3f92853ef13c82
```
Comment (abigailboyer) [inline seats/index.html:49]: and delete all of these merge lines too
